src/algebra/big_operators.lean

```diff
@@ -550,6 +550,14 @@ end
 
 end semiring
 
+section semimodule
+variables [semiring β] [add_comm_monoid γ] [semimodule β γ]
+
+lemma mul_sum' : b • s.sum h = s.sum (λx, b • h x) :=
```

Comment:
`smul_sum`?

Reply:
Looks like github goofed up when showing me which stuff to review. Ignore the preceding comment please.